### DISCUSSION TITLE
.NET 与 .NET FrameWork 反射的差异与疑问

### DISCUSSION THREAD

**Discussion by RickyHe7** [BYJRK/DotNet-Discussions#201] [C# 技术交流]
项目中需要将原来的.NET Framework框架升级成.NET ，但是发现原来的反射不能用了
具体的情况是：WPF GUI反射一个dll A 调用其中的某些方法, 但是如果dll A的方法依赖于dll B ,就会报找不到dll B的错误
在网上找到一个类似的问题，最后解决了，博客地址如下：
https://samcragg.wordpress.com/2017/06/30/resolving-assemblies-in-net-core/
但对于这个方案怎么实现的，.NET和framework框架下的反射有什么差异还是不太清楚

**Comment by SlimeNull** [comment]
你想问的是，自动加载程序集依赖吧？ 如果你要反射的程序集已经加载到你的程序中了，那它的依赖理应也被加载了吧。
